tests/ApiCore/Tests/Unit/GapicClientTraitTest.php

```diff
@@ -666,6 +667,7 @@ public static function getClientDefaults()
             'serviceName' => 'test.interface.v1.api',
             'clientConfig' => __DIR__ . '/testdata/test_service_client_config.json',
             'descriptorsConfigPath' => __DIR__.'/testdata/test_service_descriptor_config.php',
+            'gcpApiConfigPath' => __DIR__.'/../testdata/test_service_grpc_config.json',
```

Comment:
Seems like the path has an extra `/..`?

Reply:
Thanks! This is why those lines are not being triggered. Fixed.